cmake/config.cmake

```diff
@@ -44,21 +44,36 @@ set(OMR_THREAD ON CACHE BOOL "Enable thread library")
 set(OMR_COMPILER OFF CACHE BOOL "Enable the Compiler")
 set(OMR_JITBUILDER OFF CACHE BOOL "Enable building JitBuilder")
 set(OMR_TEST_COMPILER OFF CACHE BOOL "Enable building the test compiler")
+set(OMR_LLJB OFF CACHE BOOL "Enable building LLJB")
 
 set(OMR_GC ON CACHE BOOL "Enable the GC")
 set(OMR_GC_TEST ${OMR_GC} CACHE BOOL "Enable the GC tests.")
 
 set(OMR_USE_NATIVE_ENCODING ON CACHE BOOL
 	"Indicates that runtime components should use the systems native encoding (currently only defined for z/OS)"
 )
-## OMR_COMPILER is required for OMR_JITBUILDER and OMR_TEST_COMPILER
+## OMR_COMPILER is required for OMR_JITBUILDER, OMR_TEST_COMPILER and OMR_LLJB
 if(NOT OMR_COMPILER)
 	if(OMR_JITBUILDER)
 		message(FATAL_ERROR "OMR_JITBUILDER is enabled but OMR_COMPILER is not enabled")
+	else()
+		if(OMR_LLJB)
+			message(FATAL_ERROR "OMR_LLJB is enabled but OMR_COMILER and OMR_JITBUILDER are not enabled")
+		endif()
 	endif()
 	if(OMR_TEST_COMPILER)
 		message(FATAL_ERROR "OMR_TEST_COMPILER is enabled but OMR_COMPILER is not enabled")
 	endif()
+	if(OMR_LLJB)
```

Comment:
Actually, this line would not be reached because the check on L58 would cause an abort due to enabling JitBuilder but not the Compiler. Check on L60 would cause an abort when both Compiler and LLJB is enabled but JitBuilder is disabled. I'll remove that check.